packages/opentelemetry-cloud-trace-exporter/test/exporter.test.ts

```diff
@@ -90,6 +92,38 @@ describe('Google Cloud Trace Exporter', () => {
         return {} as any;
       });
 
+      sinon.stub(protoloader, 'loadSync');
+
+      sinon.replace(
+        grpc,
+        'loadPackageDefinition',
+        (): grpc.GrpcObject => {
+          traceServiceConstructor = sinon.spy(
+            (host: string, creds: grpc.ChannelCredentials) => {}
+          );
+          /* tslint:disable-next-line:no-any */
+          const def: any = {
+            google: {
+              devtools: {
+                cloudtrace: {
+                  v2: {},
+                },
+              },
+            },
+          };
+          // Replace the TraceService with a mock TraceService
+          def.google.devtools.cloudtrace.v2.TraceService = class MockTraceService
+            implements TraceService {
+            /* tslint:disable-next-line */
```

Comment:
Is it possible to just disable the casing check? No worries if not, but just curious if there is something like disable-next-line:member-variable-must-be-lowerCamelCase

Reply:
Yea good idea - after a bit of digging the following will work: 
```
/* tslint:disable-next-line:variable-name */
```